content/authoring/guidelines/sample-tests.md

```diff
@@ -9,27 +9,27 @@ next: /authoring/guidelines/submission-tests/
   - Sample tests, in addition to the kata description, can clarify handling of edge cases, inputs which should be handled in some exceptional way, or errors.
-  - Test cases provided by author show how to use testing framework and assertion libraries to the users who'd like to add their own tests.
-  - One or two test cases with large inputs can serve as a hint on performance requirements, so users can see early that their solution is too slow or times out.  They can be initially commented out or disabled in some other way, with a note that they can be enabled if user wants to verify performance of their solution.
+  - Test cases provided by the author show how to use the testing framework and assertion libraries to the users who'd like to add their own tests.
+  - One or two test cases with large inputs can serve as a hint on performance requirements, enabling users to check that their solution meets efficiency standards before submitting. These tests can be initially disabled with a note that they can be enabled if user wants to verify performance of their solution.
 - **Sample tests should be fixed, predictable, and repeatable.** Avoid sample tests which are random or change between the runs.
-- Initial tests should compile correctly and it should be possible to run them, but they are allowed to fail.
+- Initial tests should compile and execute fully without crashes or other non-test suite errors. Initially, users should expect clear, test suite-generated assertion failures for incomplete deliverables described in the instructions.
 - **Test cases from sample tests should appear also in submission tests**, possibly as a part of fixed tests.
-- If possible, **sample tests should be easy to reproduce in user's local environment.** Many users work on their solutions with their local IDE, and it's very helpful for them to have all required information available. Preloaded snippet is especially problematic for them because it's not directly visible and it makes it difficult to reproduce sample tests locally. If your sample tests rely on anything from preloaded code, consider relevant guidelines related to [description][authoring-guidelines-description] and [preloaded code][authoring-guidelines-preloaded].
+- If possible, **sample tests should be easy to reproduce in the user's local environment.** Many users work on their solutions in their local IDE and it's very helpful for them to have all required information available. The preloaded code file is especially problematic for them because it's not directly visible which makes it difficult to reproduce sample tests locally. If your sample tests rely on anything from preloaded code, consider relevant guidelines related to [description][authoring-guidelines-description] and [preloaded code][authoring-guidelines-preloaded].
```

Comment:
```suggestion
- If possible, **sample tests should be easy to reproduce in the user's local environment.** Many users work on their solutions in their local IDE and it's very helpful for them to have all the required information available. The preloaded code file is especially problematic for them because it's not directly visible which makes it difficult to reproduce sample tests locally. If your sample tests rely on anything from preloaded code, consider relevant guidelines related to the [description][authoring-guidelines-description] and [preloaded code][authoring-guidelines-preloaded].
```